Dockerfile

```diff
@@ -1,8 +1,18 @@
 # Start runtime image,
-FROM amsterdam/python:3.9-buster
+FROM amsterdam/python:3.9-slim-buster
+
+RUN apt-get update \
+ && apt-get autoremove -y \
+ && apt-get install --no-install-recommends -y \
+        libpq-dev \
+        python-dev \
+        gcc \
+        git
```

Comment:
eingelijk zie ik niet zoveel meerwaarde in een `slim-buster` base image pakken, en daar nog tools installeren. In plaats van een gedeelde laag (geen extra filesystem grootte) heb je nu een custom laag die steeds opgebouwd moet worden en nergens mee gedeeld wordt. Dan lekker de oude base image behouden.